docs/user_guide/expand_path.rst

```diff
@@ -71,6 +71,141 @@ If your session start time is present in your data path, you can indicate this f
 `1989 C standard format codes`_ for datetimes. For example, ``"{session_start_time:%Y-%m-%d}"`` will match
 ``"2021-01-02"`` and evaluate it to ``datetime.datetime(2021, 1, 2)``.
 
+Example Usage
+-----------------------
+
+Below are some full examples of how this feature can be used on some organizational patterns taken from real datasets.
+
+Example 1: `Allen Institute Visual Coding Dataset <https://registry.opendata.aws/allen-brain-observatory/>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Allen Institute's Visual Coding dataset contains, among other data, motion-corrected videos of each
+experimental session, with the directory structure shown below.
+
+.. code-block:: bash
+
+    allen-brain-observatory/
+    ¦   visual-coding-2p/
+    ¦   +-- ophys_movies/
+    ¦   ¦   +-- ophys_experiment_645691416.h5
+    ¦   ¦   +-- ophys_experiment_<session_id>.h5
```

Comment:
I have mixed feelings about including `<information that gets inserted inside angle braces>`

I used to do it all the time because it makes sense to developers or people familiar with programming

But this doc in particular is for people that may never have coded in their life, and I've personally encountered situations where they interpret the angle braces literally

I'd be fine if the code block here is just an abbreviated view of the file storage pattern using ~3 real file name examples; the f-string aspect becomes clear in the subsequent block